### PR TITLE
Test and Review #2

### PR DESCRIPTION
**Pull request by zonesoldier**
Please review and test.

### CODE REVIEW

File: js/card.js [hunk 1]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
 ,[548,100,0,0,52,5,51,5,312,20,311,20,0]
 ,[549,100,0,0,52,5,51,5,302,30,301,30,0]
 ,[550,100,0,"Increase the probability of causing Sleep by an additional 2%.",155,10,60,5,0]
+,[551,3,"Bangungot",0,77,20,79,15,151,30,153,30,154,30,159,30,0]
+,[552,3,"Bakonawa","For Non-Monk classes, increase physical damage against Boss monsters by 15%.<br/>For Monk classes, increase physical damage against Boss monsters by 10%.",77,20,79,-35,0]
+,[553,3,"Buwaya",0,77,20,79,-35,97,7,0]
+,[554,2,"Bungisngis","MaxHP + 5%, MaxSP + 5%.<br/>For every refine after +5, MaxHP + 1%, MaxSP + 1%.",15,5,16,5,0]
```
Comment (Hatfun) [inline js/card.js]: No need for `"MaxHP + 5%, MaxSP + 5%.<br/>"`, it will be written with n_tok 15 and 16 effects

File: js/card.js [hunk 2]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
...
+,[555,2,"Engkanto","Increase damage inflicted upon Poison Property enemies by 25%.<br/>Ignore 25% of Plant monster defense.",0]
```
Comment (Hatfun) [inline js/card.js]: Can't we use n_tok 45 like it's done for the Anacondaq card for the first portion of the effect? Something like
`,[555,2,"Engkanto","Ignore 25% of Plant monster defense.",45,25,0]
`

File: js/card.js [hunk 3]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
...
+,[556,1,"Manananggal","20% more damage against Undead monsters with Heal.<br/>Receive 3% more healing when someone uses Heal or Sanctuary on you.",0]
+,[557,4,"Mangkukulam","MaxSP + 10%.",16,10,0]
```
Comment (Hatfun) [inline js/card.js]: No need for any custom description, since the MaxSP part will be written by n_tok 16

File: js/card.js [hunk 4]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
...
+,[558,2,"Tikbalang","MATK + 10.<br/><BR><b>*Warning*</b>MATK Bonus not implemented yet<b>*Warning*</b>Increases the damage of Wind property magical attacks on targets by 5%.<br/>[Refine Rate +7 or higher]<br/>Add another 5% damage with Wind Magic.<br/>[Refine Rate +9 or higher]<br/>Add another 5% damage with Wind Magic.",0]
+,[559,7,"Tiyanak","CRIT + 7 against DemiHuman, Brute and Fish Race monsters.",0]
```
Comment (Hatfun) [inline js/card.js]: I'm pretty sure we can use the `n_tok[110+n_B[2]]` to add a CRIT against specific races.
Example for Assaulter (Critical Rate + 7 on DemiHuman monster):
`,[223,1,"Assaulter",0,70,10,117,7,0]`

**Comment by zonesoldier** [comment]
Changes made, closing pull request.

**Pull request closed without merging**